apps/meteor/app/livechat/server/methods/getAgentOverviewData.ts

```diff
@@ -18,9 +18,7 @@ declare module '@rocket.chat/ui-contexts' {
 
 Meteor.methods<ServerMethods>({
 	async 'livechat:getAgentOverviewData'(options) {
-		methodDeprecationLogger.warn(
-			'The method "livechat:getAgentOverviewData" is deprecated and will be removed after version v7.0.0. Use "livechat/analytics/agent-overview" instead.',
-		);
+		methodDeprecationLogger.method('livechat:getAgentOverviewData', '7.0.0', ' Use "livechat/analytics/agent-overview" instead.');
```

Comment:
Move this to another PR

Reply:
Got it :+1: Moved it here https://github.com/RocketChat/Rocket.Chat/pull/30421